### PR TITLE
AMI.js visualizations

### PR DESCRIPTION
**Pull request by y-ng**
- install ami and three
- add '3d' renders of nifti images
- add ami gui
- add mesh
- updated ui pages to use new visualizations (images in frontend channel)

### CODE REVIEW

File: src/app/patient-info/page.tsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import Image from 'next/image';
+import dynamic from 'next/dynamic';
 import PatientViewNavBar from '../navbar/navbar-patient'
 import InfoPanel from '../info-panel/info-panel'
 import './patient-info.css'
 
-var img = require('../../assets/tumor.jpg');
+const VisualizationComponent = dynamic(
```
Comment (zoeyz101) [inline src/app/patient-info/page.tsx:7]: my personal preference is that we put functions inside of the main function on a tsx page

Comment (zoeyz101) [reply src/app/patient-info/page.tsx:7]: otherwise i would move it to a different file

Comment (zoeyz101) [reply src/app/patient-info/page.tsx:7]: i think this with the dynamic component is standard for next tho so ignore this but just fyi general thoughts

**Review by zoeyz101 — APPROVED** [review]
so exciting!! thanks for getting it working ying :)

File: src/app/patient-info/page.tsx (hunk 1 above)
Comment (y-ng) [reply src/app/patient-info/page.tsx:7]: ok yeah makes sense (had to use `dynamic` to get the page components rendering in the right order for the ami stuff to work but that's probably a good idea in general)

**Pull request merged**